packages/disputer/hardhat.config.js

```diff
@@ -0,0 +1,17 @@
+const { getHardhatConfig } = require("@UMA/common");
```

Comment:
this shows how easily each package can pull its hard hat config from common. all the common configs and tasks come along for free and we simply override the relevant components for this package.